Source/WebCore/page/LocalFrame.h

```diff
@@ -384,6 +387,7 @@ class LocalFrame final : public Frame {
 
     float m_pageZoomFactor;
     float m_textZoomFactor;
+    ScrollbarMode m_scrollingMode { ScrollbarMode::Auto };
```

Comment:
It's a bit confusing that `scroll**ing**Mode` is a `Scroll**bar**Mode`. Should we just use a new enum?

Reply:
Oh but I see there's prior art.

Reply:
Yeah, prior art indeed, but very confusing. Led to many compile errors.